Tests/FunctionalTests/TestRig/Sources/FTRMultiFingerSwipeGestureRecognizerViewController.xib

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There were some warnings of constraints failure. I fixed it by removing the width constraint on the gestureRecognizerBox. This is what the xib file looks like after the changes:
```
<?xml version="1.0" encoding="UTF-8"?>
<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="11762" systemVersion="16D32" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES">
    <device id="retina3_5" orientation="portrait">
        <adaptation id="fullscreen"/>
    </device>
    <dependencies>
        <deployment identifier="iOS"/>
        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="11757"/>
        <capability name="Aspect ratio constraints" minToolsVersion="5.1"/>
        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
    </dependencies>
    <objects>
        <placeholder placeholderIdentifier="IBFilesOwner" id="-1" userLabel="File's Owner" customClass="FTRMultiFingerSwipeGestureRecognizerViewController">
            <connections>
                <outlet property="gestureRecognizedLabel" destination="m5i-4d-AdD" id="78x-UO-IcO"/>
                <outlet property="view" destination="i5M-Pr-FkT" id="sfx-zR-JGt"/>
            </connections>
        </placeholder>
        <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
        <view clearsContextBeforeDrawing="NO" contentMode="scaleToFill" id="i5M-Pr-FkT">
            <rect key="frame" x="0.0" y="0.0" width="320" height="480"/>
            <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
            <subviews>
                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="No gesture recognized yet" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="m5i-4d-AdD">
                    <rect key="frame" x="58.5" y="20" width="203" height="21"/>
                    <accessibility key="accessibilityConfiguration" identifier="gestureRegonizedLabel"/>
                    <fontDescription key="fontDescription" type="system" pointSize="17"/>
                    <nil key="textColor"/>
                    <nil key="highlightedColor"/>
                </label>
                <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="zcB-0i-jxD">
                    <rect key="frame" x="16" y="49" width="288" height="288"/>
                    <color key="backgroundColor" red="0.72336632013320923" green="0.72336632013320923" blue="0.72336632013320923" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
                    <accessibility key="accessibilityConfiguration" identifier="gestureRecognizerBox">
                        <bool key="isElement" value="YES"/>
                    </accessibility>
                    <gestureRecognizers/>
                    <constraints>
                        <constraint firstAttribute="width" secondItem="zcB-0i-jxD" secondAttribute="height" multiplier="1:1" id="i0G-ma-sr8"/>
                    </constraints>
                </view>
            </subviews>
            <color key="backgroundColor" red="1" green="1" blue="1" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
            <constraints>
                <constraint firstItem="m5i-4d-AdD" firstAttribute="centerX" secondItem="i5M-Pr-FkT" secondAttribute="centerX" id="1uI-UM-EUu"/>
                <constraint firstItem="zcB-0i-jxD" firstAttribute="leading" secondItem="i5M-Pr-FkT" secondAttribute="leading" constant="16" id="AL5-K2-iDW"/>
                <constraint firstAttribute="trailing" secondItem="zcB-0i-jxD" secondAttribute="trailing" constant="16" id="GTF-4T-BHI"/>
                <constraint firstItem="zcB-0i-jxD" firstAttribute="top" secondItem="m5i-4d-AdD" secondAttribute="bottom" constant="8" id="egh-u8-1Db"/>
                <constraint firstItem="m5i-4d-AdD" firstAttribute="top" secondItem="i5M-Pr-FkT" secondAttribute="top" constant="20" id="trU-xW-VTY"/>
            </constraints>
            <point key="canvasLocation" x="34.5" y="53.5"/>
        </view>
    </objects>
</document>
```